### PR TITLE
Bug 950045 - [Email] Canceling the 'Setting Up Account' screen doesn't c...

### PR DESCRIPTION
**Pull request by rsajdok**
...ancel operation, will progress regardless; can stack multiple instances

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=09ecff5011f68ed4901f8be9318352b88d39ed72

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9c4628530a9082806c3979241237879e4ed480ca

File: apps/email/js/cards/setup_progress.js [hunk 1]
```diff
@@ -184,7 +186,11 @@ return [
           if (err) {
             this.onCreationError(err, errDetails);
           } else {
-            this.onCreationSuccess(account);
+            if (this.createCanceled === true) {
```
Comment (jrburke) [inline apps/email/js/cards/setup_progress.js]: `if (this.createCanceled)` is enough for these types of boolean checks.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f9d1aa9085e7201a2860f993012063fe08890e86

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#nwywff9zRyKhH7PdGs1aiA

The pull request failed to pass integration tests. It could not be landed, please try again.

**Pull request merged**